coding/compressed_bit_vector.cpp

```diff
@@ -230,6 +230,27 @@ unique_ptr<CompressedBitVector> CompressedBitVectorBuilder::FromBitGroups(
   return make_unique<SparseCBV>(setBits);
 }
 
+// static
+unique_ptr<CompressedBitVector> CompressedBitVectorBuilder::FromCBV(CompressedBitVector const & cbv)
```

Comment:
You already have conflicts as Subtract() is in new-search now. Please, implement DeepCopy() virtual method instead of this static FromCBV().